app/controllers/api/v1/local_authorities_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class LocalAuthoritiesController < Api::V1::ApplicationController
+      before_action :set_cors_headers, if: :json_request?
+
+      skip_before_action :authenticate
+
+      def show
+        @local_authority = LocalAuthority.find_by(subdomain: params[:subdomain])
+
+        if @local_authority
+          respond_to(:json)
+        else
+          send_not_found_response
+        end
+      end
+
+      def send_not_found_response
+        render json: { message: "Unable to find record" },
+               status: :not_found
+      end
```

Comment:
Do you know that Rails will try to render a JSON error if that was the original format? This is what you'll get for an ActiveRecord::RecordNotFound error: 
``` json
{"status":404,"error":"Not Found"}
```

This would allow you to change the controller to:
``` ruby
module Api
  module V1
    class LocalAuthoritiesController < Api::V1::ApplicationController
      before_action :set_cors_headers, if: :json_request?
      skip_before_action :authenticate

      def show
        @local_authority = LocalAuthority.find_by!(subdomain: params[:subdomain])
        respond_to(:json)
      end
    end
  end
end
```